src/Microsoft.DotNet.Darc/DarcLib/IAssetLocationResolver.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.DarcLib;
+
+public interface IAssetLocationResolver
```

Comment:
Do we follow some kind of rule about when we should create a separate file for the interface, and when we add the interface in the same file as the implementation? In this case it's separate, while in `CoherencyUpdateResolverFactory` it's in the same file

Reply:
Ah, no rule. I put small interfaces together but so I will put this together too